.github/workflows/go.coverage.yml

```
@@ -22,6 +22,7 @@ jobs:
 
       - name: Test With Coverage
         run: |
+          go install github.com/fatih/faillint@latest || true
```

`|| true` would ignore install errors, which is probably not what we want.

```suggestion
          go install github.com/fatih/faillint@latest
```

---

ok 👍 fixed